OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -25,6 +25,12 @@ public class WithSpriteTurretInfo : ConditionalTraitInfo, IRenderActorPreviewSpr
 		[Desc("Sequence name to use")]
 		[SequenceReference] public readonly string Sequence = "turret";
 
+		[Desc("Custom palette name")]
+		[PaletteReference] public readonly string Palette = null;
```

Comment:
This needs to be `[PaletteReference("IsPlayerPalette")]` or the linter will get confused